tests/support/parser/__init__.py

```diff
@@ -332,7 +332,7 @@ def parse_args(self, args=None, values=None):
                         os.path.basename(fpath).startswith('test_'):
                     self.options.name.append(fpath)
                     continue
-                self.exit(status=1, msg='\'{}\' is not a valid test module'.format(fpath))
+                self.exit(status=1, msg='\'{}\' is not a valid test module\n'.format(fpath))
```

Comment:
Can you help me understand this change? I don't know what adding a newline here fixes.

Reply:
Without the newline then the error message is not on a line by itself and the command prompt is left on the same line as the error message.